src/it/scala-2.12/org/apache/james/gatling/JmapIT.scala

```diff
@@ -31,9 +31,10 @@ abstract class JmapIT extends GatlingFunSpec {
     server.stop()
   }
 
-  protected def scenario(scenarioFromFeeder: FeederBuilder => ScenarioBuilder) = {
+  protected def scenario(scenarioFromFeeder: (FeederBuilder, FeederBuilder) => ScenarioBuilder) = {
```

Comment:
I don't like to rely on the fact people know what the two expected feeders are. Could you define types?
Like UserFeeder(Builder) & RecipientsFeeder(Builder)?

Reply:
fixed